ops/model.py

```diff
@@ -1629,9 +1630,8 @@ def _validate_read(self):
         if self._backend.app_name == self._entity.name:
             # minions can't read local app databags
             raise RelationDataAccessError(
-                "{} is not leader and cannot read its own application databag".format(
-                    self._backend.unit_name
-                )
+                f"{self._backend.unit_name} is not leader and cannot read its own "
+                "application databag"
```

Comment:
Same as above, plus I believe Ruff likes to have consistent leading f's in implicitly concatenated strings even when not required so we'll hit that in the formatter:

```suggestion
                f'{self._backend.unit_name} is not leader and cannot read its own '
                f'application databag'
```